packages/story-editor/src/components/panels/design/layer/layerList.js

```diff
@@ -69,7 +69,7 @@ const ReorderableLayer = memo(function ReorderableLayer({
     <Fragment key={id}>
       <LayerSeparator
         groupId={element.groupId}
-        nestedOffset={nestedOffset}
+        nestedOffset={Boolean(nestedOffset)}
```

Comment:
Unrelated to this PR but was making noise in the console - prop is expected to be a Boolean